### DISCUSSION TITLE
log's topics is null

### DISCUSSION THREAD

**Discussion by amncjf** [filecoin-project/lotus#10902] [General Q&A]
https://api.calibration.node.glif.io/rpc/v1 has obtained the receipt of transaction 0x53b5686d0309e3617e02c8e3f2fe8a5facc14672d746afd948a2db8eb29f292b, where the logIndex is 7 and the topics of the log are null. Why? topics are always not null

**Comment by TippyFlitsUK** [comment]
Please use the issue templates provided to submit new tickets @amncjf!
I would also suggest asking this in https://github.com/filecoin-project/ref-fvm
Many thanks 🙏

**Comment by amncjf** [comment]
I use new issue templates to resubmit the issue, also gave the issue to https://github.com/filecoin-project/ref-fvm
Thanks🤝
